docs/architecture.md

```diff
@@ -1,103 +1,172 @@
+
+This allows you to have a lower accuracy offline model as fallback to account for internet outages, this ensures your device never becomes fully unusable.
+
+Several different STT (Speech To Text) Plugins are avaliable for use.  OVOS provides a number of public servers using the [ovos-stt-plugin-server](https://github.com/OpenVoiceOS/ovos-stt-plugin-server) plugin and are hosted by OVOS contributors.  No additional configuration is required.
+
+[OVOS Supported STT Plugins](https://github.com/orgs/OpenVoiceOS/repositories?q=ovos-stt-plugin&type=all&language=&sort=)
+
+**TODO** Fix link to how to change stt
+
+[Changing STT Plugin](stt-plugins.md)
+
+### WakeWord Plugins
 
-### Message
-A Message consists of a json payload, it contains a `type` , some `data` and a `context`. 
+A WakeWord is what OVOS uses to activate the device.  By default `Hey Mycroft` is used by OVOS.  Like other things in the OVOS ecosystem, this is configurable.
 
-The `context` is considered to be metadata and might be changed at any time in transit, the `context` can contain anything depending on where the message came from, and often is completely empty. 
+[OVOS Supported WakeWord Plugins](https://github.com/orgs/OpenVoiceOS/repositories?q=ovos-ww-plugin&type=all&language=&sort=)
 
-You can think of the message `context` as a sort of session data for a individual interaction, in general messages down the chain keep the `context` from the original message, most listeners (eg, skills) will only care about `type` and `data`. 
+**TODO** Fix link to change ww
 
-ovos-core uses the message `context` to add metadata about the messages themselves, where do they come from and what are they intended for.
+[Changing the WakeWord](change-ww.md)
 
-#### Sources
+#### HotWords
 
-ovos-core injects the context when it emits an utterance, this can be either typed in the `ovos-cli-client` or spoken via STT service
+OVOS allows you to load any number of hot words in parallel and trigger different actions when they are detected
 
-STT will identify itself as `audio`
+each hotword can do one or more of the following:
 
-ovos-cli-client will identify itself as `debug_cli`
+  - trigger listening, also called a **wake_word**
+  - play a sound
+  - emit a bus event
+  - take ovos-core out of sleep mode, also called a **wakeup_word** or **standup_word**
+  - take ovos-core out of recording mode, also called a **stop_word**
 
-`mycroft.conf` contains a `native_sources` section you can configure to change how the audio service processes external requests
+**TODO** fix link to adding HotWords
 
-#### Destinations
+[Setting and adding HotWords](setting-HotWords.md)
 
-Output capable services are the cli and the TTS
+### VAD Plugins
 
-The command line is a debug tool, it will ignore the `destination`
+VAD Plugins serve to detect when you are actually speaking to the device, and when you quit talking.
```

Comment:
Consider adding a sentence that says typically a user will not change the VAD plugin.

Reply:
depending on your microphone this might actually be common, and depending on platform too

the default VAD is not great either, silero is absolutely recommended but not a default dependency